pandas/core/generic.py

```diff
@@ -9571,6 +9571,8 @@ def where(
         cond : bool {klass}, array-like, or callable
             Where `cond` is {cond}, keep the original value. Where
             {cond_rev}, replace with corresponding value from `other`.
+            if `cond` {klass} is less in size than `other`, the default bool
```

Comment:
I think this would be better demonstrated in the `Example` section below

Reply:
@mroeschke What do you think now?